src/__myproject__/_export/clickhouse_export.py

```diff
@@ -37,20 +38,27 @@ def get_secrets(dbutils: DBUtils):
 
 # COMMAND ----------
 
-@dp.notebook_function("%odapfeatures.clickhouse.host%", get_secrets)
-def clickhouse_connection(clickhouse_host: str, secrets: dict):
-    return ClickhouseClient(
-        clickhouse_host,
-        user=secrets["user"],
-        password=secrets["pass"]
-    )
+@dp.notebook_function("%odapfeatures.clickhouse.host%", "%odapfeatures.clickhouse.port%")
+def get_clickhouse_address(host: str, port: int):
+    return {
+        "host": host,
+        "port": port,
+    }
+
+# COMMAND ----------
+
+def execute_clickhouse_query(query: str):
+    response = requests.post(
+        f"http://{get_clickhouse_address.result['host']}:{get_clickhouse_address.result['port']}/",
+        params={"user": get_secrets.result["user"], "password": get_secrets.result["pass"], "default_format": "JSON"},
+        data=query)
+    return ast.literal_eval(response.content.decode("utf8") or "{}").get("data")
```

Comment:
why ast necessary?

Reply:
I'm parsing a bytes result and I found this to be a way to turn it into a dict. Is there a better way?